app/src/main/res/layout/updates_tab_fragment.xml

```diff
@@ -7,6 +7,7 @@
   </data>
 
   <FrameLayout
+    android:id="@+id/updates_tab_container"
```

Comment:
It would be great to document this in the PR description so that it can give context when one checks the changelog for this change(I tend to look at the git history of a line to see the PR description that added it, if not documented locally).